### PR TITLE
chore(cli): remove unused function

### PR DESCRIPTION
**Pull request by eventualbuddha**
I believe that technically this function is part of the public API via `dioxus_cli::cli::cfg::parse_public_url`. However, I don't see any evidence that this function is actually used inside or outside this crate. The last use within the crate was removed in January 2022 (9bb10c65a3cbabef127f6af5c0d1eadab1d01531).

### CODE REVIEW

**Review by Demonthos — APPROVED** [review]
Thank you! (This is technically a breaking change, waiting for a second review before merging)

**Comment by Esales301080** [comment]
Good

**Pull request merged**
